docs/training.md

```diff
@@ -1,186 +1,259 @@
 
-# Run the random forest model
-trained_random_forest = hcai_trainer.random_forest()
-trained_random_forest.roc_curve_plot()
+# View the ROC and PR plots
+trained_lr.roc_plot()
+trained_lr.pr_plot()
 
+# View the ROC and PR plots
+trained_random_forest.roc_plot()
+trained_random_forest.pr_plot()
 ```
 
-### Go further using utility methods
+#### Comparison plots
 
-The `healthcareai.common.model_eval.tsm_classification_comparison_plots` method plots ROC curves or PR curves of one or more trained models to directly comparing models.
+The `healthcareai.common.model_eval.tsm_classification_comparison_plots` method plots ROC curves or PR curves of one or more trained models to directly compare models. It's arguments are:
 
--   **Return**: a plot.
--   **Arguments**:
-    - **plot_type** (str): 'ROC' (default) or 'PR' 
-    - **trained_supervised_model** (list | TrainedSupervisedModel): a single or list of TrainedSupervisedModels
-    - **save** (bool): True to save the plot.
+- **plot_type** *(str)*: 'ROC' (default) or 'PR' 
+- **trained_supervised_models** *(list | TrainedSupervisedModel)*: a single or list of TrainedSupervisedModels 
+- **save** *(bool)*: Save the plot to a file
 
-Example code:
+##### Example code
 
 ```python
-# Create a comparison ROC plot multiple models
-hcaieval.tsm_classification_comparison_plots(
-    trained_supervised_model=[trained_random_forest, trained_logistic_regression],
+# Create a list of all the models you just trained that you want to compare
+models_to_compare = [trained_knn, trained_lr, trained_random_forest]
+
+# Create a ROC plot that compares them.
+tsm_plots.tsm_classification_comparison_plots(
+    trained_supervised_models=models_to_compare,
     plot_type='ROC',
     save=False)
 
-# Create a comparison PR plot multiple models
-hcaieval.tsm_classification_comparison_plots(
-    trained_supervised_model=[trained_random_forest, trained_logistic_regression],
+# Create a PR plot that compares them.
+tsm_plots.tsm_classification_comparison_plots(
+    trained_supervised_models=models_to_compare,
     plot_type='PR',
     save=False)
 ```
 
+## Step 5: Save a Model
+
+After you have trained a model you are happy with, you can save that model for later use.
+
+On any instance of a `TrainedSupervisedModel` (that the trainer returns), use the `.save()` method to save the model.
+
+Models will be saved with a timestamp and algorithm name so it is easy to find which one you want.
+
+```python
+# Save the model
+trained_random_forest.save()
+```
+
+
 ## Full example code
 
-Note: you can run (out-of-the-box) from the healthcareai-py folder:
+Note: you can run this out-of-the-box from the healthcareai-py folder:
 
 ```python
-from healthcareai import SupervisedModelTrainer
 import pandas as pd
-import time
+
+import healthcareai.trained_models.trained_supervised_model as tsm_plots
+import healthcareai.common.database_connections as hcai_db
+from healthcareai.supvervised_model_trainer import SupervisedModelTrainer
```

Comment:
typo: should be supervised_model_trainer
the same typo occurs in test_top_factors.py

similar typo for advanced supervised trainer (advanced_supvervised_trainer) occurs throughout the project